stable/syncthing/2.1.0/test_values.yaml

```diff
@@ -0,0 +1,36 @@
+# Default values for Syncthing.
+
+image:
+  repository: syncthing/syncthing
+  pullPolicy: IfNotPresent
+  tag: 1.15
+
+strategy:
+  type: Recreate
+
+services:
+  main:
+    port:
+      port: 8384
+
+persistence:
+  config:
+    enabled: false
+    emptyDir: false
+  storage:
+    enabled: false
+    emptyDir: false
+
+## TrueCharts Config
+
+appVolumeMounts:
+  config:
+    enabled: true
+    emptyDir: true
+    setPermissions: true
+    mountPath: "/var/syncthing/"
+  storage:
+    enabled: true
+    emptyDir: true
+    setPermissions: true
+    mountPath: "/var/syncthing/storage"
```

Comment:
Is this needed?
Users can add custom storage now, so we don't need to add mountpoints for all their different storage wishes (media/userdata/downloads) anymore.

Reply:
It's not. My rationale behind it was that by default, the folder Syncthing uses for the config is the same default folder for new syncable folders that get added, and having a `config` volume that also gets defaulted to for new sync folders didn't seem good. 

In hindsight I think it makes more sense to just have a `storage` volume that goes to /var/syncthing/ for the config and default folders, and to let users add non-volume storage themselves.

Reply:
This can be resolved if you agree, I've committed the changes I mentioned above.